src/Comms.h

```diff
@@ -1,19 +1,44 @@
 #pragma once
 
 #include <Arduino.h>
+#include <Ethernet.h>
+#include <EthernetUdp.h>
+#include <map>
+#include <string>
+
+byte mac[] = {0xDE, 0xAD, 0xBE, 0xEF, 0xDA, 0x02};
+int port = 42069;
+IPAddress ip(10, 0, 0, 42);
```

Comment:
You should move these inside the namespace, and make them const

Reply:
Also, every teensy has a unique Mac address built into the hardware. There's a function you can use to get the Mac address of the teensy, and you should use that to set this array. This is how it was done in the old code: 
<img width="619" alt="image" src="https://user-images.githubusercontent.com/13842083/147394602-2c7ff5ef-b184-4e8e-a64c-6750fd999209.png">